### DISCUSSION TITLE
Would this be helpful?

### DISCUSSION THREAD

**Discussion by samuelgoto** [fedidcg/FedCM#548] [General]
TIL that github apparently has this section that enables discussions within a repo! i'm curious to see if this can be helpful!!!

WDYT?

has anyone tried this before?

**Comment by kdenhartog** [comment]
I've used them ocassionally for standards discussion in the past. As with any messaging feature it's how it's used more than anything. This has been useful for longer ongoing architecture discussions this has been similar to what a mailing list can serve. As usual with these types of discussions it can lead to thread drift, but that's somewhat better here since we can split out specific points to issues as we come away with action items. I think it could be beneficial here, especially when first seeking consensus of a explainer or other big change that requires a bit longer feedback cycles.

**Comment by TallTed** [comment]
GitHub's "discussions" are not and do not enable discussions.

These are Q&A, more akin to StackOverflow and its cousins than to any threaded discussion, but lacking handling of any of the lessons learned by those other sites/cousins.

You get one Question, and unlimited Answers, with unlimited single-layer comments on each Answer. There is no further threading. There is no good way to see all comments in chronological order (such as would be needed to see all comments since your last viewing).

There are many other weaknesses/failures in GitHub's implementation of "discussions". I do not recommend their use for anything.
